src/main/java/org/myrobotlab/lang/py/LangPyUtils.java

```diff
@@ -289,26 +290,31 @@ public Map<Integer, String> buildPython(StringBuilder content, Boolean includeHe
 
       content.append(newPythonContent);
 
-      MetaData serviceType = ServiceData.getMetaData(si.getName(), si.getSimpleName());
-      Map<String, ServiceReservation> peers = serviceType.getPeers();
+      MetaData.getConfigType(filename);
+
+      // static getDefault routes by type
+      Plan plan = ServiceConfig.getDefault(Runtime.getPlan(), si.getName(), si.getSimpleName());
+      ServiceConfig sc = plan.get(si.getName());
+      Map<String, Peer> peers = sc.getPeers();
 
       // FIXME - do "indent"
       boolean firstTime = true;
-      if (maxDepth == null || maxDepth < currentDepth) {
+      if ((maxDepth == null || maxDepth < currentDepth) && peers != null) {
         for (String peer : peers.keySet()) {
-          ServiceReservation sr = peers.get(peer);
-          ServiceInterface peerSi = Runtime.getService(sr.actualName);
+          Peer sr = peers.get(peer);
+          String noWorky = "noWorky"; // had to fix
```

Comment:
huh?  What's going on here?

Reply:
We've abandoned LangPyUtils I believe
it was doing some weird things with  peers, if we abandoned it, I don't want to spend time fixing it